packages/components/src/inputs/src/SearchField.module.css

```diff
@@ -0,0 +1,57 @@
+.hop-SearchField  {
+    --hop-SearchField-gap: var(--hop-space-stack-xs);
+    --hop-SearchField-default-inline-size: 20rem;
+    --hop-SearchField-color: var(--hop-neutral-text-weak);
+    --hop-SearchField-padding: var(--hop-space-inset-md);
+
+    display: flex;
+    flex-direction: column;
+    gap: var(--hop-SearchField-gap);
+    inline-size: var(--hop-SearchField-default-inline-size);
+}
+
+.hop-SearchField--fluid  {
+    inline-size: 100%;
```

Comment:
you should be setting an internal variable here and 100% should be a component variable.